druid-shell/src/platform/mac/window.rs

```diff
@@ -156,7 +163,7 @@ impl WindowBuilder {
             handler: None,
             title: String::new(),
             menu: None,
-            size: Size::new(500.0, 400.0),
+            size: Size::new(0., 0.),
```

Comment:
See issue #1543: because of this change, windows don't appear at all on Mac if their size are not explicitly set.